changelog.md

```diff
@@ -67,6 +67,11 @@
 
     + Add RTL support (Alexander Krotov, #5551).
 
+  * HTML reader:
```

Comment:
Leave off the changelog change for now.  I will gather these together from the git log eventually, but since I look at the log since the last modification of the changelog, modifications now will confuse the process...

Reply:
This explanation should go in the git commit message for the actual change.

Reply:
Okay ill add it to the commit when I squash